samples/compression/lz4/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_LZ4=y
+CONFIG_MINIMAL_LIBC=y
```

Comment:
Just a nit: no other sample specifies `CONFIG_MINIMAL_LIBC=y` explicitly, so I would remove it from here as well.